hazelcast/src/main/java/com/hazelcast/core/MemberLeftException.java

```diff
@@ -57,13 +58,14 @@ private void writeObject(ObjectOutputStream out) throws IOException {
         out.defaultWriteObject();
```

Comment:
Client-side exceptions are instantiated on the client-side by `ClientExceptionFactory` that only delivers the exception type, message and stack trace elements, not using java serialization.